src/config/dynamic-theme-fixes.config

```diff
@@ -561,6 +561,16 @@ INVERT
 
 ================================
 
+bbs.thinkpad.com
+club.lenovo.com.cn
```

Comment:
`.cn` Not needed.

Reply:
It's actually needed in https://club.lenovo.com.cn/think or the individual forum post page like https://club.lenovo.com.cn/thread-5954537-1-1.html